src/utils/node.ts

```diff
@@ -0,0 +1,42 @@
+/*
+ * Copyright © 2023 Lisk Foundation
+ *
+ * See the LICENSE file at the top-level directory of this distribution
+ * for licensing information.
+ *
+ * Unless otherwise agreed in a custom licensing agreement with the Lisk Foundation,
+ * no part of this software, including this file, may be copied, modified,
+ * propagated, or distributed except according to the terms contained in the
+ * LICENSE file.
+ *
+ * Removal or modification of this copyright notice is prohibited.
+ */
+import { Port } from '../types';
+import { isPortAvailable } from './network';
+import { execAsync } from './process';
+
+const INSTALL_LISK_CORE_COMMAND = 'npm i -g lisk-core';
+
+export const installLiskCore = async (): Promise<string> => execAsync(INSTALL_LISK_CORE_COMMAND);
+
+// Export for testing
+export const isAppVersion3 = (version: string): boolean =>
+	version.length > 0 && version.startsWith('3');
```

Comment:
This method serves no purpose. Please remove this.
Also, this is an incorrect check because it'd successfully verify `30.0.0`.

Reply:
Applied.